### PR TITLE
Pretend to fix a bug

### PR DESCRIPTION
**Pull request by Fausto95**
## Detailed purpose of the PR

> :wave: Write here :wink:

### Testing strategy

- [ ] I ensured my code is tested: 
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

**Comment by VincentCATILLON** [comment]
No, that not resolves the issue

File: src/components/catalog-section.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ class CatalogSection extends React.PureComponent<Props> {
     const suffix =
       (item && item.universalRef && item.universalRef.replace(/_/g, '-')) || `${index}-placeholder`;
 
-    return `catalog-section-${sectionRef || testID}-item-${suffix}`;
+    return `catalog-section-${index}-${sectionRef || testID}-item-${suffix}`;
```
Comment (chrisdugne) [inline src/components/catalog-section.js]: il y a deja `index` dans `suffix`

Comment (Fausto95) [reply src/components/catalog-section.js]: Seulement pour les placeholders

Comment (VincentCATILLON) [reply src/components/catalog-section.js]: i understand what you're going to do, but do it in suffix please :)

Comment (VincentCATILLON) [reply src/components/catalog-section.js]: should be `catalog-section-${sectionRef || testID}-item-${index}-${suffix}`

**Comment by Fausto95** [comment]
Closing in favor of https://github.com/CoorpAcademy/coorpacademy/pull/14223 since is doing the fix on the server

**Pull request closed without merging**